src/api/taxonomyApi.js

```diff
@@ -15,34 +15,49 @@ import {
 
 const baseUrl = apiResourceUrl(`/taxonomy/v1/`);
 
-async function queryResources(articleId, accessToken, language) {
+async function queryResources(articleId, accessToken, language, contentType) {
   const response = await fetch(
-    `${baseUrl}queries/resources?contentURI=urn:article:${articleId}&language=${language}`,
+    `${baseUrl}queries/resources?contentURI=urn:${contentType}:${articleId}&language=${language}`,
```

Comment:
Kunne kalla det contentId i stedet for articleId over alt.

Reply:
:+1: